src/command_modules/azure-cli-interactive/azclishell/configuration.py

```diff
@@ -13,7 +13,7 @@
 
 SELECT_SYMBOL = {
     'outside': '#',
-    'query': '?',
+    'query': '??',
```

Comment:
But the "use my last result" ? is only ever 1) at the beginning or 2) right after an equal sign.  You should be able to distinguish these (and this is why the unit tests of this are so important!)